helpers.js

```diff
@@ -184,6 +202,33 @@ exports.moveAllFilesInDir = (dirToSearch, dirToMoveTo) => {
   })
 }
 
+exports.addDependenciesToStore = deps => {
```

Comment:
These are the helpers I mentioned to add packages to the store. And then when the project is finished building files run the installAllPackages helper. Something to consider however is that if something breaks not all packages will install. We need to come up with a way to perhaps add the packages name and a stable version number to the package.json file.

Reply:
It is possible to add in a catch if something fails, to just skip it and move on to the next one? So in the end, the user gets a message like "x packages failed to install" but it still installs all of the rest. 

Possibly keep a record of failed installs so once everything else is installed, we can map back over the failed, downgrading the version number.